package com.task5.exception

data class ApiError(
    val errorCode: String, //task.not.found
    val description: String,
)
